### ISSUE TITLE
Unable to open/read a hive

### ISSUE THREAD

**Issue by Space1230** [minacle/chntpw#1]
On macos M2, I got the following error while trying to create an iso using UUP dump.
```
Creating boot.wim...
openHive(): read error: : Inappropriate ioctl for device
chntpw: Unable to open/read a hive, exiting..
```
This error seems to be consistent as I tried multiple other builds of Windows Cumulative Updates.

**Comment by minacle** [comment]
Thank you for leaving your issue. However, with my configuration, I was unable to reproduce the problem you experience. If possible, please provide the ZIP file provided by UUP Dump.

**Comment by ynyyn** [comment]
It seems that similar issue happened on me, I got similar error

```shell
Creating boot.wim...
openHive(): read error: : Inappropriate ioctl for device
chntpw: Unable to open/read a hive, exiting..
[WARNING] Ignoring extended attributes of 1 files
"ISODIR/sources/boot.wim" original size: 350696 KiB
Using LZX compression with 10 threads
Archiving file data: 941 MiB of 941 MiB (100%) done
"ISODIR/sources/boot.wim" optimized size: 344459 KiB
Space saved: 6236 KiB

Exporting Windows 10 Professional to install.wim...
[ERROR] Destination image description contains illegal characters
ERROR: Exiting with error code 24:
       An invalid parameter was given.
Failed to export Windows 10 Professional to install.wim

```

when trying to create an ISO for **arm64** Windows via `./uup_download_macos.sh` for [Feature update to Windows 10, version 22H2 (19045.3155) arm64](https://uupdump.net/selectlang.php?id=3ef377ed-a40e-4603-b179-bb556bbf0f20), with MacOS 12.6.5 @ Apple M1 Pro.

However, this is the second build I tried. The first time I got this error is with ([Windows 11, version 22H2 (22621.1848) arm64](https://uupdump.net/download.php?id=3ef377ed-a40e-4603-b179-bb556bbf0f20)). Soon after I got this error, I noticed there was an alert on UUPDump download page saying that

> (Windows 11, version 22H2 or later)
> This build requires using Windows 10, version 2004 or later for the ISO to be properly created. **Scripts for other platforms will output a broken ISO.**

I am not sure whether it is related but anyway later then I turned to the `Windows 10, version 22H2` one who does not has such alert. However, it still failed with this error. 

I am not familiar with how UUP Dump works, thus I have no idea what went wrong. So I just leave what I can tell here and hope it may be helpful.

**Comment by minacle** [comment]
This issue has been resolved in #4.
